.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
+ - Version [e.g. 22]
```

Comment:
I'd be curious about `Device` for desktop platforms as well, since in rare cases, certain OS / browser combinations can behave differently based on microphone access and hardware 🤯 what do you think about merging "Desktop" and "Smartphone" into a single "Platform" category?

Maybe we could also add an "Additional hardware [eg. were you using headphones, an external microphone, wireless speakers, etc]" section?

Reply:
Sounds great!

Reply:
Addressed in [ce59282](https://github.com/mozilla/voice-web/pull/2773/commits/ce592825183754b359cb56b5f296ec7a71050067)